src/video/cocoa/SDL_cocoawindow.m

```diff
@@ -2116,6 +2116,15 @@ take effect properly (e.g. setting the window size, etc.)
     }
 }
 
+int
+Cocoa_FlashWindow(_THIS, SDL_Window *window, Uint32 flash_count)
+{ @autoreleasepool
+{
+    /* Note that this is app-wide and not window-specific! */
+    [NSApp requestUserAttention:NSInformationalRequest];
```

Comment:
There isn't a way to specify the exact number of 'flashes' (icon bounces) in macOS as far as I know but `NSCriticalRequest` can be used to make the icon bounce continuously instead of just once. This comment is probably out of the scope of this specific PR but maybe it makes sense to change SDL_FlashWindow's external API to more closely follow a common denominator across operating systems? For example in my own stuff I've implemented similar functionality with a `continuous` bool parameter, which I've implemented in both macOS and Windows.

Reply:
A continuous bool makes sense to me - I'm surprised any OS has it down to a specific number of flashes in the first place. X11 and Wayland are indifferent either way so I'm up for whatever here.

Reply:
I suppose an enum (`FLASH_ONCE`, `FLASH_CONTINUOUS`) is nicer than a bool – but in any case I feel a bit bad for cluttering this self-contained PR. :)